JSDemos/Demos/DataGrid/GroupSummaries/description.md

```diff
@@ -1 +1,17 @@
-This example illustrates the use of&nbsp;group data summaries (calculated totals in&nbsp;each group). To&nbsp;configure a&nbsp;group summary, declare the **groupItems** array within the **summary** object. This demo also illustrates the component&rsquo;s ability to&nbsp;position group summary items. By&nbsp;default, they are displayed in&nbsp;the group row&rsquo;s header, but you can shift them to&nbsp;a&nbsp;specific column (the **alignByColumn** property) or&nbsp;even move them to&nbsp;the group footer (the **showInGroupFooter** property).
\ No newline at end of file
+To configure group summaries, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[groupItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/) array with summary configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.
+
+### Position a Group Summary
+
+Group summaries are displayed in parentheses after the group caption. You can reposition them as follows:
+
+* Display a summary in a column          
+You can align a summary to its corresponding column (see the Sale Amount and Total Amount columns). To do this, enable the [alignByColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#alignByColumn) property. If you want to align a summary to a different column, specify the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+* Display Summary Values within Group Footers       
+You can make a summary item display its values within group footers. To do that, enable the [showInGroupFooter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInGroupFooter) property. Summary items will appear within their corresponding columns (see the Total Amount column). If you want to display summaries in another column, specify the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+### Format Summary Values
+To format summary values, use the [valueFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#valueFormat) property. This demo applies the *"currency"* format to summary values calculated against the Total Amount and Sale Amount columns. If you also want to add custom text to summary values, use the [displayFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#displayFormat) property. This demo sets this property for summaries of *"sum"* and *"count"* types.
```

Comment:
demo?

Reply:
Yes, demo - it is available here: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/GroupSummaries/React/Light/

Reply:
ok thanks